src/components/layout/layout.js

```diff
@@ -206,8 +207,8 @@ const Layout = ({
             FooterNav = <Footer />
             break
         case 'landing-page':
-            Navigation = <Nav hide_signup_login={true} no_language={true} />
-            FooterNav = <Footer />
+            Navigation = <NavInterim no_button={true} no_languageSwitcher={no_languageSwitcher} />
```

Comment:
since we are using new version of JS and React, there is no need of passing **true** to these props,
just add no_button and no_language_switcher here